projects/midnight-commander.org/package.yml

```diff
@@ -51,5 +51,10 @@ build:
       - --with-screen=ncurses
 
 # FIXME: hard to test interactive tools
-test: test "$(mc --datadir | cut -d ' ' -f1 | cut -d':' -f1)" = "{{prefix}}/etc/mc/"
+test:
+  env:
+    TERM: vt100  # needs $TERM set to run
```

Comment:
Question is, should brewkit set this? Possibly should be set to `dumb`?

Reply:
I had a vague memory that things didn't like `dumb`. At the present time, this hack is only in two packages; that said, if this is a recent change in brewkit, more `curses` programs may display this behavior.

Reply:
At v1 I began sanitizing build/test env so that it is about as bare as possible.

`TERM` might be something we should let through or set, but I'm not really sure about what it means.

Reply:
likely should not sanitize:

https://www.baeldung.com/linux/term-environment-variable

Reply:
i would include `LC_*` vars as things we shouldn't modify.